.github/workflows/ci.yml

```diff
@@ -56,6 +56,9 @@ jobs:
           key: ${{ runner.os }}-cmake-${{ hashFiles('gradle/libs.versions.toml', 'app/build.gradle.kts', 'app/src/main/cpp/CMakeLists.txt') }}
           restore-keys: ${{ runner.os }}-cmake-
 
+      - name: Install Build Tools
+        run: echo "y" | ${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager --install "build-tools;35.0.0-rc1"
```

Comment:
Why do we need to install this manually?

Reply:
Apparently preview SDKs have a different license than those pre-installed ones https://github.com/FooIbar/EhViewer/actions/runs/7940239629/job/21681254875